src/pages/trade/HorizontalPanel.tsx

```diff
@@ -272,8 +274,17 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen }: ElementProps) => {
     ]
   );
 
+  const dispatch = useAppDispatch();
+  const onCancelOrders = useCallback(
+    (marketId: string) => {
+      dispatch(openDialog({ type: DialogTypes.CancelPendingOrders, dialogProps: { marketId } }));
+    },
+    [dispatch]
+  );
```

Comment:
This FN can probably live within the `<PotentialPositionCard>` so it doesn't need to be added here and in the Portfolio. Each card already uses a selector and knows it's own `marketId`

Reply:
Would reduce amount of prop drilling